### PR TITLE
ZK-4423: Kanban missing options to listen to portallayout onPortalMov…

### PR DESCRIPTION
**Pull request by plijyfes**
…e without affecting the UI

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -3416,6 +3416,8 @@ F86-ZK-4235.zul=A,E,datefmt,library-property
 ##zats##F95-ZK-3289.zul=A,E,Datebox
 ##zats##F95-ZK-4418.zul=A,E,Filedownload,ContentType
 ##zats##F95-ZK-4418-2.zul=A,E,Filedownload,ContentType
+##zats##F95-ZK-4423.zul=A,E,Portallayout,onPortalMove,MVVM
```
Comment (plijyfes) [inline zktest/src/archive/test2/config.properties:3419]: add case for MVC

**Review by scribetw — CHANGES_REQUESTED** [review]
Add MVC usage zul case

File: zktest/src/org/zkoss/zktest/test2/F95_ZK_4423Composer.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		Clients.log(evevt.getName());
+	}
+
+	@Listen("onPortalMove = #pl2")
+	public void onPortalMove2(PortalMoveEvent evevt) {
+		Clients.log(evevt.getName());
+	}
+
+	@Listen("onPortalDrop = #pl3")
+	public void onPortalDrop3(PortalDropEvent evevt) {
```
Comment (plijyfes) [inline zktest/src/org/zkoss/zktest/test2/F95_ZK_4423Composer.java:55]: typo and add portal move logic

**Review by DevChu — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**